src/main/java/projectLibraryPartOne/service/LibraryServiceImpl.java

```diff
@@ -0,0 +1,91 @@
+package projectLibraryPartOne.service;
+
+import projectLibraryPartOne.entity.Book;
+
+import java.util.ArrayList;
+
+public class LibraryServiceImpl implements LibraryService {
+
+    private ArrayList<Book> listBooks;
+
+    public LibraryServiceImpl() {
+        this.listBooks = new ArrayList<>();
+    }
+
+    public ArrayList<Book> getListBooks() {
```

Comment:
we work at the interface level (List< Book>)

Reply:
fixed

Reply:
also, not see fix